n_request.c

```diff
@@ -522,9 +523,9 @@ J *noteTransactionShouldLock(J *req, bool lockNotecard)
     // Debug
     if (suppressShowTransactions == 0) {
         if (responseJSON[strlen(responseJSON)-1] == '\n') {
-            _Debug(responseJSON);
+            NOTE_C_LOG_INFO(responseJSON);
         } else {
-            _Debugln(responseJSON);
+            NOTE_C_LOG_INFO(responseJSON);
```

Comment:
This else can be removed.

Reply:
looks more like the `if` should be removed.